dashboard/app/views/levels/editors/fields/_csa_fields.html.haml

```diff
@@ -16,6 +16,43 @@
     - maze_value = @level.serialized_maze ? JSON.pretty_generate(@level.serialized_maze) : ''
     = f.text_area :serialized_maze, value: maze_value, rows: 20, class: 'input-block-level'
 
+-# This element exists instead of the _encrypted_examples element in order to facilitate level urls
+-# instead of project urls in the example solutions. This is needed because Java Lab has mini-app
+-# types, such as neighborhood, which are highly dependent on their level definition and do not have
+-# a standalone type. However, we can't use the same method used by Maze or other puzzle level
+-# types. Those send the solution to every client's browser. If we used that method, the solution
+-# would be available to view the solution from the browser. Instead, we encrypt the solution url
+-# and put the level in a hidden script. Only verified teachers will be able to view the level.
```

Comment:
I think it is fine to say "only verified teachers" for now, but I just wanted to confirm my understanding that the current plan is to provide this access only to pilot teachers, and find a new solution (such as "exemplar mode", similar to "start mode"), before launch.

Reply:
Yes. I've added a task on our backlog: https://codedotorg.atlassian.net/browse/CSA-496